akka-javasdk/src/main/scala/akka/javasdk/impl/consumer/ConsumerEffectImpl.scala

```diff
@@ -20,17 +20,13 @@ import akka.javasdk.consumer.Consumer
 private[impl] object ConsumerEffectImpl {
   sealed abstract class PrimaryEffect extends Consumer.Effect {}
 
-  final case class ProduceEffect[T](msg: T, metadata: Option[Metadata]) extends PrimaryEffect {
-    def isEmpty: Boolean = false
-  }
+  final case class ProduceEffect[T](msg: T, metadata: Option[Metadata]) extends PrimaryEffect {}
 
-  final case class AsyncEffect(effect: Future[Consumer.Effect]) extends PrimaryEffect {
-    def isEmpty: Boolean = false
-  }
+  final object ConsumedEffect extends PrimaryEffect {}
 
-  case object IgnoreEffect extends PrimaryEffect {
-    def isEmpty: Boolean = true
-  }
+  final case class AsyncEffect(effect: Future[Consumer.Effect]) extends PrimaryEffect {}
+
+  case object IgnoreEffect extends PrimaryEffect {}
```

Comment:
we probably don't need this one? could use ConsumedEffect for `ignore()` right away?

Reply:
yeap, removing it